parser/preprocessed/SqlLexer_gen.cpp

```diff
@@ -8,8 +8,8 @@
 
 #define FLEX_SCANNER
 #define YY_FLEX_MAJOR_VERSION 2
-#define YY_FLEX_MINOR_VERSION 6
-#define YY_FLEX_SUBMINOR_VERSION 0
+#define YY_FLEX_MINOR_VERSION 5
```

Comment:
Sure. My machine has Flex 2.5.37. I can work with you to generate the pre-processed files for version 2.5.38+. 

How about the requirements for Bison version?